providers/shared/components/baseline/id.ftl

```diff
@@ -6,7 +6,7 @@
     [#return formatSegmentResourceId(SEED_RESOURCE_TYPE)]
 [/#function]
 
-[#function getBaselineComponentIds links cmk="cmk" ssh="ec2KeyPair" oai="originAccessId"]
+[#function getBaselineComponentIds links storage="bucket" cmk="cmk" ssh="ec2KeyPair" oai="originAccessId"]
```

Comment:
When adding new optional parameters its probably better to add it to the end instead of the front. Functions use positional arguments so to prevent updates across the codebase its probably safer

Reply:
@roleyfoley I considered that but after doing a quick search in the gen3 repo found there were no current calls to it with non-defaults, and with only the one plugin provider at the moment I thought we'd be safe. Can update it if you feel we should.

Reply:
Moved dd96aec